### ISSUE TITLE
Readme doc

### ISSUE THREAD

**Issue by dindoliboon** [renisac/CIF3-pwsh#1]
Hi Michael,

Thanks for sharing this module!

In README.md there is an example that has:
```PowerShell
Get-CIF3Indicator -Confidence 8 -ResultSet 500
```

But that returns an error.
```
Get-CIF3Indicator : A parameter cannot be found that matches parameter name 'ResultSet'.
```

The parameter ResultSet should be ResultSize instead.
```PowerShell
Get-CIF3Indicator -Confidence 8 -ResultSize 500
```

**Comment by mdavis332** [comment]
Nice catch @dindoliboon (also, sweet Avatar :)! Do you want to submit a PR for this change?

**Comment by dindoliboon** [comment]
Sure thing, I'll submit a PR for the change.

**Comment by mdavis332** [comment]
Thanks! Fixed by merging #3
